src/main/java/billennium/tests/controller/web/rest/QuizController.java

```diff
@@ -24,27 +24,31 @@
 @RequiredArgsConstructor
 public class QuizController {
 
-    public static final String ROOT_MAPPING = "/api/quiz";
+    public static final String ROOT_MAPPING = "/api/quiz"; //TODO to samo co w QuestionController
 
     private final QuizService quizService;
     private final QuestionService questionService;
 
 
-    @RequestMapping(value = "/{quiz_id}/questions", method = RequestMethod.GET)
+    @RequestMapping(value = "/{quiz_id}/questions", method = RequestMethod.GET) //TODO wszystkie value w
+    // requestMapping
+    // tez do controllerConstatnst - bo tutaj np quiz_id sie powtarza a może być z jednego constantsam Prosba do
+    // wszystkich metod w controllerach: uzywaj @GetMapping/@PostMapping itp zamiast @RequestMapping z okresleniem
+    // metody
```

Comment:
a co ty na to by np. wykorzysta: javax.ws.rs tam mamy po prostu @GET @POST.....?

Reply:
szczerze to nigdy sie nie spotkałem z tym w komercyjnym projekcie.